app/services/instagram-api-client.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import ENV from "dependency-injection-demo/config/environment";
 
-export default Ember.Object.extend({
+export default Ember.Service.extend({
```

Comment:
Make sure your service is under the `Ember.Service` namespace and move it into a services folder while you're at it!